engines/director/resource.cpp

```diff
@@ -312,6 +313,15 @@ Archive *DirectorEngine::loadEXE(const Common::Path &movie) {
 		for (uint i = 0; i < versions.size(); i++) {
 			Common::WinResources::VersionInfo *info = exe->getVersionResource(versions[i]);
 
+			Common::String gameName = info->hash["FileDescription"];
+			Common::String versionInfo = Common::String::format("v%d.%d.%dr%d", info->fileVersion[0], info->fileVersion[1], info->fileVersion[2], info->fileVersion[3]);
+
+			if (!Director::FallbackBlacklist::contains(gameName)) {
```

Comment:
For the sake of simplifying the code, please remove this check. Then you do not need this ugly class.